demisto_sdk/commands/generate_outputs/generate_descriptions/generate_descriptions.py

```diff
@@ -0,0 +1,289 @@
+import itertools
+import logging
+import math
+import os
+import os.path
+import threading
+import time
+from typing import Dict
+
+import requests
+
+from demisto_sdk.commands.common.tools import get_yaml, print_error, write_yml
+
+CREDENTIALS = 9
+
+old_merge_environment_settings = requests.Session.merge_environment_settings
+
+# Globals
+STOP_SEQS = ["\n", "\ncontextPath:"]
+CURRENT_PROMPT = "contextPath: Guardicore.Incident.source_asset.labels\ndescriptionMessage: The source assets labels for the Incident." \
+                 "\ncontextPath: Something.Incident.reenrich_count\ndescriptionMessage: The number of re-enrichments made on an incident." \
+                 "\ncontextPath: VeryBigWord.Endpoint.vm.orchestration_details.orchestration_obj_id" \
+                 "\ndescriptionMessage: The orchestration object identifier for an Endpoint's virtual machine." \
+                 "\ncontextPath: AAA.Endpoint.status\ndescriptionMessage: The status of an Endpoint (off/on)." \
+                 "\ncontextPath: DDDdddRRR.Endpoint.is_on_alert\ndescriptionMessage: Whether the endpoint is on or not." \
+                 "\ncontextPath: Blabla.Endpoint.active\ndescriptionMessage: Whether the endpoint is on or off." \
+                 "\ncontextPath: Guardicore.Endpoint.nics.is_cloud_public\ndescriptionMessage: Whether the Endpoint's NIC is in the public cloud or not." \
+                 "\ncontextPath: Activity.Endpoint.active\ndescriptionMessage: Whether the Endpoint is on or off." \
+                 "\ncontextPath: IDID.Endpoint.host_orchestration_id\ndescriptionMessage: The orchestration object identifier for the Endpoint's host." \
+                 "\ncontextPath: BrublaBRO.Endpoint.is_on\ndescriptionMessage:  Whether the Endpoint is on or not." \
+                 "\ncontextPath: WellDone.Endpoint.metadata.InventoryAPI.report_source\ndescriptionMessage:  The Endpoint's InvestoryAPI report source." \
+                 "\ncontextPath: SomethingElse.Endpoint.bios_uuid\ndescriptionMessage:"
```

Comment:
What is this? why is this the init value

Reply:
This is what tells ai21 how we want to generate the outputs, it needs a few examples to learn. 
https://en.wikipedia.org/wiki/One-shot_learning

Without it it won't know the type of task or way we want it to generate results.

Reply:
Got it. Could you add a comment above that those are init values for examples to AI21 to learn from?

Reply:
Yep :+1: